flags.go

```diff
@@ -50,10 +50,15 @@ var (
 		Name:  "tlsverify",
 		Usage: "Use TLS and verify the remote",
 	}
+	flOverCommit = cli.IntFlag{
+		Name:  "overcommit, oc",
+		Usage: "OverCommit to apply on resources",
+		Value: 105,
```

Comment:
If undercommit is actually desired we definitely should change the flag to not call it overcommit. 

However, I'm not really sure I understand your motivation though, if I don't want a container to be consume all of the box's resources, we should instead advertise less than the maximum memory/cpu instead of undercommiting right? Since lower resource usage can also lead to more binpacked containers to fill the gaps.